historian-server/src/main/java/com/hurence/webapiservice/http/api/grafana/GrafanaHurenceDatasourcePluginApiImpl.java

```diff
@@ -103,6 +103,7 @@ private JsonObject buildGetMetricsParam(SearchRequestParam request) {
     public final static String SAMPLING_ALGO_JSON_PATH = "/sampling/algorithm";
     public final static String BUCKET_SIZE_JSON_PATH = "/sampling/bucket_size";
     public final static String REQUEST_ID_JSON_PATH = "/request_id";
+    public final static String AGGREGATION_JSON_PATH = "/agreg";
```

Comment:
Here agreg iw hat the client will have to enter in request. That is why ti is preferable to not use shortcuts.

Reply:
ok